README.md

```diff
@@ -93,7 +118,10 @@ The [people.json file](https://github.com/cncf/people/blob/main/people.json) lis
     }
 ```
 
-Project names must exactly match [the landscape listing](https://landscape.cncf.io/card-mode?project=hosted).  Category names must exactly match the list above.
+TODO Fix broken link here OR update this statement
+Project names must exactly match [the landscape listing](https://landscape.cncf.io/card-mode?project=hosted).  
+TODO Are Category names missing?
```

Comment:
Should these be TODO comments, or is this section more of an FAQ or Troubleshooting section?